kolibri/plugins/learn/assets/src/views/CustomChannelRenderer.vue

```diff
@@ -0,0 +1,202 @@
+<template>
+
+  <div>
+    <KContentRenderer
```

Comment:
My suggestion here was to copy the relevant parts of the `HTML5RendererIndex.vue` implementation into here, so that you have more direct access to the iframe (also in this way, there are now two Hashi instances, one in this component, the other inside the HTML5 renderer component).

Reply:
> so that you have more direct access to the iframe

Yeah, that would be really helpful if this component provided more access to the `iframe` and the "kolibri side" of the hashi, just like in `HTML5RendererIndex`. I believe this will make it easier to access the theming set within the custom channel HTML5 and pass it to the modal holding the ContentRenderer.

Reply:
Actually, I see below the point that there are now two hashis in play here. So now I'm wondering which hashi I would need to access to get the theme!